packages/integration-karma/test/native-shadow/children/index.spec.js

```diff
@@ -0,0 +1,36 @@
+import { createElement } from 'lwc';
+import Slottable from 'x/slottable';
+
+describe('children', () => {
+    function expectOneSpanChild(slot) {
+        expect(slot.children.length).toEqual(1);
+        expect(slot.childElementCount).toEqual(1);
+        expect(slot.firstElementChild && slot.firstElementChild.tagName).toEqual('SPAN');
+        expect(slot.lastElementChild && slot.lastElementChild.tagName).toEqual('SPAN');
+        expect(slot.firstChild && slot.firstChild.tagName).toEqual('SPAN');
+        expect(slot.lastChild && slot.lastChild.tagName).toEqual('SPAN');
+        expect(slot.hasChildNodes()).toEqual(true);
+        expect(slot.childNodes.length).toEqual(1);
```

Comment:
I don't think it worth creating a brand new test file for this. There are already those tests that can be improved:

- https://github.com/salesforce/lwc/blob/master/packages/integration-karma/test/shadow-dom/ParentNode-properties/ParentNode.children.spec.js
- https://github.com/salesforce/lwc/blob/master/packages/integration-karma/test/shadow-dom/ParentNode-properties/ParentNode.childElementCount.spec.js
- https://github.com/salesforce/lwc/blob/master/packages/integration-karma/test/shadow-dom/Node-properties/Node.firstChild.spec.js
- https://github.com/salesforce/lwc/blob/master/packages/integration-karma/test/shadow-dom/Node-properties/Node.lastChild.spec.js
- https://github.com/salesforce/lwc/blob/master/packages/integration-karma/test/shadow-dom/Node-properties/Node.hasChildNodes.spec.js

Reply:
These tests are broken up by APIs (`children`, `firstChild`, `lastChild`) etc. I think it would be a lot of extra code to explode my tests out into the same test but for 5 different files. Plus it would make the tests run longer.

Also, I like having tests that address specific issues/WIs. We can point to this exact test and say "this proves that the issue is fixed." If it's spread across multiple files then that's harder to reason about.